server/odc-service/src/main/java/com/oceanbase/odc/service/regulation/risklevel/RiskLevelService.java

```diff
@@ -117,6 +117,11 @@ public Optional<RiskLevel> findById(@NonNull Long id) {
         return riskLevelRepository.findById(id).map(this::entityToModel);
     }
 
+    @SkipAuthorize("internal usage")
+    public Optional<RiskLevel> findRawById(@NonNull Long id) {
```

Comment:
You can write a method named `nullSafeGet()`, refer to `public UserEntity nullSafeGet(String accountName)`

Reply:
`    private RiskLevel entityToModel(RiskLevelEntity entity) {
        RiskLevel model = riskLevelMapper.entityToModel(entity);
        model.setApprovalFlowConfig(approvalFlowConfigService.findById(entity.getApprovalFlowConfigId()));
        return model;
    }

    private RiskLevel rawEntityToModel(RiskLevelEntity entity) {
        RiskLevel model = riskLevelMapper.entityToModel(entity);
        return model;
    }`

The main difference lies here.`model.setApprovalFlowConfig(()`